recipes/scherbelberg/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "scherbelberg" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scherbelberg-{{ version }}.tar.gz
+  sha256: 8ff1665653b60110549896bf808dc963b972641af8c21ca46650eb0353d0a03c
+
+build:
+  number: 0
+  noarch: python
```

Comment:
Upstream uses `scripts` so this cannot be no arch anymore: https://github.com/pleiszenburg/scherbelberg/blob/ee3615984ab3ca797e19ba0434c20b92f3aff611/setup.py#L105

Reply:
@BastianZim I am also the upstream maintainer: Shall I simply remove the line `scripts=[],`? I am actually using `entry_points` ... `scripts` is probably a relic from an old project template.

Reply:
Ahh great, that makes it much simpler! Yes, that would be the best approach. 

But we’d need a new version with that too or you can apply it as a patch here - whatever you find easier.

Reply:
I'll push a new release to PyPI and update this very PR. Should take 10 minutes plus CI checks ... I'll ping you when it's done.

Reply:
@BastianZim Done. CI test done, too - ready for another review.